planning_domains_api.py

```diff
@@ -41,15 +41,15 @@ def checkForDomainPath():
         USER_TOKEN = list(filter(lambda x: x.tag == 'token', installationSettings))[0].text
     return True
 
-def query(qs, qtype="GET", params={}, offline=False, format='/json'):
+def query(qs, qtype="GET", params={}, offline=False, format='/json', formalism = "/classical"):
```

Comment:
Formalism shouldn't have the leading `/`. The usage below (which adds it) is correct.